### ISSUE TITLE
Support regular expressions in EXCLUDE_SYMBOLS (Origin: bugzilla #636706)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#4040]
status NEW severity *enhancement* in component *general* for *---*
Reported in version *1.7.1* on platform *Other*
Assigned to: Dimitri van Heesch

On 2010-12-07 14:49:43 +0000, Umesh wrote:

    Would be really useful if you could extend the EXCLUDE_SYMBOLS tag to
    support regular expressions

**Comment by albert-github** [comment]
With e.g.
```
/// \file

/// a f fie
void f1();
/// a f fie
void f2();
/// a g fie
void g1();
/// a g fie
void g2();
```
and 
```
EXCLUDE_SYMBOLS = g*
```
We see already in the 1.7.1 version that there are only the `f1` and f2` functions shown.
Looking in the current documentation we see though that he given note about the wildcard is like the one for `EXCLUDE_PATTERNS` and should be adjusted.

Problem is as far as I can see a documentation problem.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/11158689/example.tar.gz)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9967

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
